main/exercise/ptest_category_ranking.class.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/* For licensing terms, see /license.txt */
+
+use Chamilo\CourseBundle\Entity\CQuizAnswer;
+use ChamiloSession as Session;
+
+/**
+ * Class PtestCategoryRanking.
+ *
+ * This class allows to instantiate an object of type QUESTION_PT_TYPE_CATEGORY_RANKING
+ * (Choose a word (or sentence) from a set of possible answers),
+ * extending the class question
+ *
+ * @author Jose Angel Ruiz (NOSOLORED)
+ *
+ * @package chamilo.exercise
+ */
+class PtestCategoryRanking extends Question
+{
+    public $typePicture = 'mcua.png';
+    public $explanationLangVar = 'PtestCategoryRanking';
+
+    /**
+     * Constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->type = QUESTION_PT_TYPE_CATEGORY_RANKING;
+        $this->isContent = $this->getIsContent();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function createAnswersForm($form)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $nb_answers is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)